Lib/ssl.py

```diff
@@ -1201,9 +1200,10 @@ def DER_cert_to_PEM_cert(der_cert_bytes):
     PEM version of it as a string."""
 
     f = str(base64.standard_b64encode(der_cert_bytes), 'ASCII', 'strict')
-    return (PEM_HEADER + '\n' +
-            textwrap.fill(f, 64) + '\n' +
-            PEM_FOOTER + '\n')
+    ss = [PEM_HEADER]
+    ss += [f[i:i+64] for i in range(0, len(f), 64)]
```

Comment:
+1 for the idea

Please don't use ``+=`` and use a self-explaining local variable name:
```
lines = [PEM_HEADER]
lines.extend(f[i:i+64] for i in range(0, len(f), 64))
lines.extend([PEM_FOOTER, ''])
return '\n'.join(lines)
```

Reply:
My first impulse was to add a similar comment. But actually this doesn't matter. All lists here are small I hope, so copying lists doesn't add much overhead. On other hand, the variant with `extend()` and generator expression add their own overheads.

1. Generator expression is slightly slower than a list comprehension.
2. `list.extend()` is special cased for tuples and list. It is slower with general iterators.
3. Calling a method is slower than executing an operator.
4. I didn't checked, but maybe `PEM_FOOTER + '\n'` is faster than `[PEM_FOOTER, '']`.

Hence the current code looks good enough to me. It is enough fast and looks enough Pythonic. If there are opportunities to enhance it, it is not worth the spent time.

Reply:
I'm sorry about variable name.  I didn't care about name for very short scope variables.

In case of performance, as serhiy said, cert size won't be arbitrary large.
And old code used temporal string for body part.  This commit didn't make anythong worse.

```
$ ./python -m perf timeit -s "import textwrap; s='a'*1000" -- 'textwrap.fill(s, 60)'
.....................
Mean +- std dev: 192 us +- 10 us

$ ./python -m perf timeit -s "import textwrap; s='a'*1000" -- 'x=[]; x+=[s[i:i+60] for i in range(0,len(s),60)]; "\n".join(x)'
.....................
Mean +- std dev: 4.66 us +- 0.06 us
```